io/eolearn/io/processing_api.py

```diff
@@ -230,3 +293,75 @@ def _extract_bands(self, eopatch, images, shape):
             bands = [np.round(band * norm, 4) for band, norm in zip(bands, norms)]
 
         eopatch[self.bands_feature] = np.asarray(bands).reshape(*shape, num_bands)
+
+    def _add_meta_info(self, eopatch):
+        eopatch.meta_info['maxcc'] = self.maxcc
+        eopatch.meta_info['time_difference'] = self.time_difference
+
+
+class SentinelHubDEMInputTask(SentinelHubInputBase):
+    ''' A processing API input task that downloads the digital elevation model
+    '''
+    def __init__(self, dem_feature, size=None, resolution=None, cache_folder=None, config=None,
+                 max_threads=None):
+        """
+        :param size: Number of pixels in x and y dimension.
+        :type size_x: tuple(int, int)
+        :param bands_feature: Target feature into which to save the downloaded images.
+        :type bands_feature: tuple(sentinelhub.FeatureType, str)
+        :param bands: An array of band names.
+        :type bands: list[str]
+        :param additional_data: A list of additional data to be downloaded, such as SCL, SNW, dataMask, etc.
+        :type additional_data: list[tuple(sentinelhub.FeatureType, str)]
+        :param maxcc: Maximum cloud coverage.
+        :type maxcc: float
+        :param time_difference: Minimum allowed time difference, used when filtering dates, None by default.
+        :type time_difference: datetime.timedelta
```

Comment:
These docs strings need to be updated / removed.

Reply:
Done.